packages/mobile/src/harmony-native/components/layout/Box/Box.tsx

```diff
@@ -1,8 +1,43 @@
 import styled from '@emotion/native'
+import { View } from 'react-native'
 
 import type { BoxProps } from './types'
 
-export const Box = styled.View<BoxProps>(
+const invalidProps = [
```

Comment:
so it was an issue when i used "as" prop on these, it would then pass all these props to those components, some of which actually expect things like "borderRadius", and they dont expect borderRadius='m', instead they expect numbers, so then i was getting errors. I think prob just good habit for us not to leak these props to the sub components?